cloud-functions-http/container/translations/shell-translations.csv

```diff
@@ -53,41 +53,41 @@ Cloud 함수의 이름은 실행하려는 메서드의 이름과 동일해야 
  注意：部署功能可能需時幾分鐘。"
 DEPLOYMENT_HINT,"The function you want to deploy is called $FUNCTION
 
-You want to set the function to triger on http requests, which means you want to use the --trigger-http flag
+You want to set the function to trigger on http requests, which means you want to use the --trigger-http flag
 
 This means you can deploy the function with:
 
-gcloud functions deploy $FUNCTION --runtime $RUNTIME --trigger-http","A função que você quer implantar é chamada $FUNCTION
+gcloud functions deploy $FUNCTION --runtime $RUNTIME --trigger-http --allow-unauthenticated","A função que você quer implantar é chamada $FUNCTION
```

Comment:
Why do we want to add the `--allow-unauthenticated` flag here?

Reply:
When you leave that out, Cloud Functions will prompt you: 
`Allow unauthenticated invocations of new function [HelloHTTP]? (y/N)?`

I figured since this is the hint already, we shouldn't leave options open, to make sure people don't get stuck.